cpp/oneapi/dal/backend/primitives/reduction/reduction_rm_cw_naive_local_dpc.cpp

```diff
@@ -0,0 +1,186 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/backend/primitives/reduction/reduction_rm_cw_dpc.hpp"
+
+#include "oneapi/dal/backend/primitives/common.hpp"
+
+namespace oneapi::dal::backend::primitives {
+
+template <typename Float, typename BinaryOp, typename UnaryOp>
+class kernel_reduction_rm_cw_naive_local {
+    using acc_t =
+        sycl::accessor<Float, 1, sycl::access::mode::read_write, sycl::access::target::local>;
+    using inp_t = const Float*;
+    using out_t = Float*;
+
+public:
+    kernel_reduction_rm_cw_naive_local(acc_t cache,
+                                       inp_t const input,
+                                       out_t const output,
```

Comment:
Use of `const` is not consistent. In some functions `inp_t const` and `out_t const` are not present, but `inp_t` and `out_t` used, for example see `reduction/reduction_rm_rw_dpc.hpp:32`. 

Maintain consistency in your own code! If `const` intentionally added to specific pointers, use this approach in all functions.

Reply:
It's kernel

Reply:
Fixed